flowlogs_reader/flowlogs_reader.py

```diff
@@ -144,6 +162,24 @@ def __init__(self, event_data, EPOCH_32_MAX=2147483647):
             ('pkt_dst_aws_service', str),
             ('flow_direction', str),
             ('traffic_path', int),
+            ('resource_type', str),
```

Comment:
The logs we have do not necessarily abide by these aws tgw headers. The case we have is a one of a kind where tgw logs are mixed with other vpc flowlogs. For example: the value for `account-id` is set to `TransitGateway` . In ideal case it would be the `resource-type` that should have been set to that value.

Reply:
The library should support mixed logs.

This PR should allow the user to do:
```python
all_flow_logs = S3FlowLogsReader(...)
no_tgw_logs = (x for x in all_flow_logs if x.account_id != 'TransitGateway')
```

This would filter out the TGW logs.